scripts/verify-release.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -eux
+
+./gradlew clean buildCodeSearchAssets buildPluginAndAssertAgentBinariesExist runPluginVerifier
```

Comment:
This script will run before pushing a tag. CI will only build and publish artifacts. Running `runPluginVerifier` is too slow to run in CI because it requires downloading multiple IDEA versions.